### DISCUSSION TITLE
How to dynamically clear the session cache

### DISCUSSION THREAD

**Discussion by WANHO-O** [waylaidwanderer/node-chatgpt-api#262] [Help]
When consecutive conversations exceed the token limit, the session fails. I want to be able to dynamically clear the session cache before a certain time according to the conversation.

**Comment by waylaidwanderer** [comment]
Are you talking about BingAIClient's jailbreak mode? If this is the case, it's not possible at the moment, but you're welcome to open a PR.

**Reply by WANHO-O** [reply]
wow, your reply is really fast.I'm talking about ChatGPTClient.（I'm new to nodejs, sorry to bother you）

**Reply by waylaidwanderer** [reply]
ChatGPTClient already has code to make sure the conversation never exceeds the model's token limit though. Maybe I'm misunderstanding your issue?

**Reply by WANHO-O** [reply]
You are right, I apologize for my stupidity and for taking up your time!

**Reply by waylaidwanderer** [reply]
All good. You can set `maxPromptTokens` along with `modelOptions.max_tokens` to a lower value if you wish to reduce your token usage.
